Gemfile

```diff
@@ -10,6 +10,7 @@ ruby '~> 2.7.1'
 # Server
 gem 'puma', '~> 5.0.4'
 gem 'rails', '>= 6.0.3.4'
+gem 'mimemagic'
```

Comment:
This is now completely removed as a dependency from this project. I've added it here to reflect that we still use the mimetype functionality of the mimemagic gem